app.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python
-from flask import Flask, render_template, Response, request, flash
-from flask_sqlalchemy import SQLAlchemy
 import mediapipe_processor
 import psycopg2
 import redis
+from flask import Flask, render_template, Response, request, flash
+from flask_sqlalchemy import SQLAlchemy
```

Comment:
The [PEP8#imports](https://peps.python.org/pep-0008/#imports) says:

+ Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
  
  Imports should be grouped in the following order:
  1. Standard library imports.
  2. Related third party imports.
  3. Local application/library specific imports.

  You should put a blank line between each group of imports.